lte/gateway/c/session_manager/test/ProtobufCreators.cpp

```diff
@@ -84,30 +84,28 @@ void create_rule_record(
 }
 
 void create_rule_record(
-    const std::string& imsi, const std::string& ip, const std::string& rule_id,
-    uint64_t bytes_rx, uint64_t bytes_tx, uint32_t teid,
+    const std::string& imsi, const std::string& ip, uint32_t teid,
+    const std::string& rule_id, uint64_t bytes_rx, uint64_t bytes_tx,
     RuleRecord* rule_record) {
-  create_rule_record(imsi, ip, rule_id, bytes_rx, bytes_tx, rule_record);
+  create_rule_record(imsi, rule_id, bytes_rx, bytes_tx, rule_record);
+  rule_record->set_ue_ipv4(ip);
```

Comment:
Welcome to sessiond world 
Since we are not using IPs on rule records anymore, you remove that parameter and cascade this to any function that creates a rule record.

Reply:
Lets do all sessiond review here https://github.com/magma/magma/pull/6232

Reply:
Removing that line produces ~8 errors in test_local_enforcer. Not sure how many of those are real vs just expecting rule record to have an ip_addr. @themarwhal  could you take a look

Reply:
Yeah let me clean up the ip field in the other PR.

Reply:
@uri200 I've addressed your comment in the other PR. :)

Reply:
@koolzz can cherrypick again into this PR.